compiler/modules/front.syntax/tests/testParserOnTestSuite.bal

```diff
@@ -37,49 +36,7 @@ function testParserOnTestSuite() returns err:Syntax|io:Error|file:Error? {
                 continue;
             }
             else {
-                SourceFile file = part.file;
```

Comment:
We should do this so that each file is a separate test as far as testerina is concerned (as we do for `front`)

Reply:
fixed